projects/aca-playwright-shared/src/api/rules-api.ts

```diff
@@ -22,12 +22,148 @@
+    if (numOfConditions > ConditionsTypes.conditions.length) {
+      numOfConditions = ConditionsTypes.conditions.length;
+    }
+    const randomActionsIndex = crypto.randomInt(0, ActionTypes.actions.length);
+    const randomAction = ActionTypes.actions[randomActionsIndex];
+    let conditionsArray = [];
+    for (let i = 0; i < numOfConditions; i++) {
+      const randomIndex = crypto.randomInt(0, ConditionsTypes.conditions.length);
+      const randomCondition = ConditionsTypes.conditions[randomIndex];
+      conditionsArray.push(randomCondition);
+    }
+    const response = await this.createRule(folderId, {
+      name: ruleName,
+      isEnabled: true,
+      actions: [randomAction],
+      conditions: {
+        inverted: false,
+        booleanMode: 'and',
+        compositeConditions: conditionsArray
+      }
+    });
+    return response;
+  }
+
+  async createRandomRuleWithMultipleActions(folderId: string, ruleName: string, numOfActions: number): Promise<Rule> {
+    if (numOfActions > ActionTypes.actions.length) {
+      numOfActions = ActionTypes.actions.length;
+    }
+    let actionsArray = [];
+    for (let i = 0; i < numOfActions; i++) {
+      const randomIndex = crypto.randomInt(0, ActionTypes.actions.length);
+      const randomAction = ActionTypes.actions[randomIndex];
+      actionsArray.push(randomAction);
+    }
+    const response = await this.createRule(folderId, {
+      name: ruleName,
+      isEnabled: true,
+      actions: actionsArray
+    });
+    return response;
+  }
+
+  async createRuleWithRandomAspects(folderId: string, ruleName: string): Promise<Rule> {
+    const response = await this.createRule(folderId, {
+      name: ruleName,
+      isEnabled: true,
+      actions: [
+        {
+          actionDefinitionId: 'add-features',
+          params: {
+            'aspect-name': 'sc:controlsAreClearance'
+          }
+        },
+        {
+          actionDefinitionId: 'add-features',
+          params: {
+            'aspect-name': 'sfdc:objectModel'
+          }
+        },
+        {
+          actionDefinitionId: 'add-features',
+          params: {
+            'aspect-name': 'sfdc:folder'
+          }
+        }
+      ]
+    });
+    return response;
+  }
+
+  async createRuleWithDestinationFolder(
+    folderId: string,
+    ruleName: string,
+    actionType: 'move' | 'copy' | 'import',
+    destinationFolderId: string
+  ): Promise<Rule> {
+    const response = await this.createRule(folderId, {
+      name: ruleName,
+      isEnabled: true,
+      actions: [
+        {
+          actionDefinitionId: actionType,
+          params: {
+            'destination-folder': destinationFolderId
+          }
+        }
+      ]
+    });
+    return response;
+  }
+}
+
 type RuleTrigger = 'inbound' | 'update' | 'outbound';
 
-export interface RuleCompositeCondition {
+export interface RuleCondition {
```

Comment:
Okay so first of all this new interface doesn't match with what API returns also you can remove it from here and just use `RuleCompositeCondition` from `aca-content`